pkg/xen-tools/xen-start

```diff
@@ -29,8 +29,10 @@ handleUnknownState() {
   else
     # Number of times we got unknown state is <= UnknownStateThreshold, so declaring the state as running
     echo running
-  fi
 
+    # Waiting for 5sec before checking domain status again in order to recover from unknown state
+    sleep 5
```

Comment:
Odd to have this sleep in this function. Can't we do the sleep in the (loop in the) caller? That would make it much easier to understand.